.gitignore

```diff
@@ -17,3 +17,4 @@ selenium-debug.log
 *.ntvs*
 *.njsproj
 *.sln
+*.js text eol=lf
```

Comment:
This should be included in the  `.gitattributes file`, not `.gitignore`

Reply:
oops so sorry, my bad! Yea now I have added it in `.gitattributes file` 
Do you need to add some other formats also like 
`*.jsx   eol=lf`
`*.json  eol=lf`
@brenoferreira